constants.py

```diff
@@ -1,12 +1,40 @@
 import os
 import numpy as np
 
-# Things anyone wishing to use this repository for their
-# own use will want to change
-MEDIA_DIR = os.path.join(
-    os.path.expanduser('~'),
-    "Dropbox (3Blue1Brown)/3Blue1Brown Team Folder"
-)
+env_MEDIA_DIR = None
+MEDIA_DIR = "#ERROR#"
+
+try:
+    env_MEDIA_DIR = os.getenv("MEDIA_DIR")
+except NameError:
+    try:
+        env_MEDIA_DIR = os.environ['MEDIA_DIR']
+    except KeyError:
+        pass
+
+if not (env_MEDIA_DIR is None):
+    MEDIA_DIR = env_MEDIA_DIR
+elif os.path.exists("media_dir.txt"):
+    with open("media_dir.txt", 'rU') as media_file:
+        MEDIA_DIR = media_file.readline()
+    if MEDIA_DIR[-1] in ['\n', '\r']:
+        raise Exception("media_dir.txt must be a single-line text file pointing at the media_dir place")
```

Comment:
why not
```python
with open("media_dir.txt", 'rU') as media_file:
    MEDIA_DIR = media_file.readline().strip()
```

Reply:
Tools like `pipenv` or `docker-compose` load env variables from `.env` files. I find this file loading complicating this already long file.
With adoption of `.env` you could expect that user can handle variables.

Reply:
1. I just see with your comment that I wrote something stupid... The MEDIA_DIR cannot end with a new line. Anyways, I'll use that `strip()` command too, because it seems to be useful.
2. Yes, but manim must accept to add this dependancy, and I don't know how to use `pipenv` nor `docker-compose`.